lib/members.ex

```diff
@@ -0,0 +1,62 @@
+defmodule GlassFactoryApi.Members do
+  @moduledoc """
+  Module which hold the GET functions of organization members.
+  """
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Members.Member
+
+  @doc """
+  List the members of a organization
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.list()
+      [ %Member{
+          name: "John Doe",
+          email: "john.doe@example.org",
+          id: 0,
+          archived: false,
+          capacity: 8.0,
+          freelancer: false,
+          joined_at: "2019-01-01"
+        }
+      ]
+  """
+  @spec list_members() :: list(Member.t())
+  def list_members() do
+    {:ok, %{status_code: 200, body: body}} = ApiClient.get("members")
+
+    body
+    |> Jason.decode!()
+    |> Enum.map(&Member.to_struct(&1))
+  end
+
+  @doc """
+  Get a member given the id
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.get("123")
+      %Member{
+         name: "John Doe",
+         email: "john.doe@example.org",
+         id: 0,
+         archived: false,
+         capacity: 8.0,
+         freelancer: false,
+         joined_at: "2019-01-01"
+      }
+  """
+  @spec get_member(String.t()) :: Member.t()
```

Comment:
I think we can cover in this spec the case where this function returns nil, something along the lines of `@spec get_member(String.t()) :: Member.t() | nil`